packages/@dcos/sorting/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Have we considered the benefits of it being an npm package vs the extra work that is to maintain that? I like the idea of it to belong to its on package, not sure if I really like it being its own package.

Reply:
Yes, I think having it as an extra pain will be more pain, but the right kind of pain. I hope that if we aggressively split out packages we will be forced to improve our tooling so that spinning up a new package will become pretty trivial. I also think someone else could use this sorting pretty well 🤷‍♂️

Reply:
`sortBy` is a great tool for sorting an array of objects by a property, but I think it doesn't quite fit our use case here. We wanted to have tiebreakers in our sorting and we also wanted to support sorting for different data types which are not only JS data types. For me, it would be a replacement for `Array.prototype.sort`, but it might be that I misunderstand the API

Reply:
I would rather see us using something like Lodash than having to maintain this in the long run.